hasher-matcher-actioner/mypy.ini

```diff
@@ -1,3 +1,8 @@
+[mypy]
+
+[mypy-threatexchange.*]
```

Comment:
Sad! I thought most of threatexchange was typed.

Reply:
yeah... brought that up in #453 we should probably revisit